eng/common/native/common-library.sh

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env bash
+
+function GetNativeInstallDirectory {
```

Comment:
As @safern mentioned, you probably won't need this folder or the cross one, so don't check them in for now and add them to the git ignore. Anything checked in should not be present on that git ignore, and anything that you do need and is checked should NOT be in the git ignore.